docs_src/design/ucr/Core-Data-Retention.md

```diff
@@ -0,0 +1,48 @@
+## Core Data Retention and Persistent Caps
+
+### Submitters
+Jim White (IOTech Systems)
+
+## Change Log
+Formerly referred to as Core Data Cache
```

Comment:
Missing `status` from the template. Can be added when updating the UCR to `approved`